assembl/static/js/app/views/message.js

```diff
@@ -1524,6 +1546,52 @@ var MessageView = Marionette.LayoutView.extend({
     this.listenToOnce(this.messageModerationOptionsView, 'moderationOptionsClose', this.onModerationOptionsClose);
   },
 
+  onDeleteMessageClick: function(ev){
+    var that = this;
+    // We could try to minimize context switching for the user, by scrolling the viewport to the message the user wants to delete, as soon as the confirmation popin opens, using this line of code:
+    // that.messageListView.scrollToMessage(that.model, false, false);
+    
+    var onSubmit = function(ev){
+      var analytics = Analytics.getInstance();
+      analytics.trackEvent(analytics.events.MESSAGE_LIKED);
+      // we could use that.model.destroy() instead, and add || method == "delete" to models/message.js::sync()
+      var message_delete_url = that.model.getApiV2Url();
+      Promise.resolve(
+        $.ajax(message_delete_url, {
```

Comment:
@swergas There was no other way of calling the DELETE route with backbone? Or was this method easier to resolve? If so, let's comment in the message model in the front-end that the backbone `Backbone.Model.destroy` method will not call this route, and instead, use the `$.ajax(url, {method: 'delete'})` method.